### ISSUE TITLE
Add additional labels to the metrics

### ISSUE THREAD

**Issue by emaincourt** [go-gorm/prometheus#46]
## Describe the feature

Hi,

As far as I know there is no way to define additional labels to attach to the exported metrics. It would be nice to allow the user to explicitly define custom labels to identify where the client metrics come from (eg. `DBStats`). Please tell me if I'm  just missing the configuration that allows to do so. Otherwise, I would be very pleased to contribute in case my request seems relevant to you.

## Motivation

Basically the limitation I'm facing at the moment with the current implementation is that when exporting metrics from multiple replicas of the same service, `db_name` is not enough to identify which metrics belong to which replica. Each replica should export metrics labeled with an identifier to make `DBStats` replica scoped.

## Related Issues

N/A

Thanks in advance
